integration-tests-cypress/support/login.ts

```diff
@@ -0,0 +1,61 @@
+import { submitButton } from '../views/form';
+import { masthead } from '../views/masthead';
+
+declare global {
+  namespace Cypress {
+    interface Chainable<Subject> {
+      login(providerName?: string, username?: string, password?: string): Chainable<Element>;
+      logout(): Chainable<Element>;
+    }
+  }
+}
+
+const KUBEADMIN_USERNAME = 'kubeadmin';
+const KUBEADMIN_IDP = 'kube:admin';
+
+// any command added below, must be added to global Cypress interface above
+
+// This will add 'cy.login(...)'
+// ex: cy.login('my-idp', 'my-user', 'my-password')
+Cypress.Commands.add('login', (provider: string, username: string, password: string) => {
```

Comment:
As a follow on, we should make these available in an npm library we can update instead of making it part of the template project directly, but this is OK for now.